locale/shine/extensions/basecommands/ruRU.json

```diff
@@ -0,0 +1,70 @@
+	"KICK_LANGUAGE": "Мат",
+	"KICK_MIC_SPAM": "Спам в микрофон",
+	"GAG": "Молчанка",
+	"GAG_TIP": "Не даёт возможности игроку использовать голосовой и обычный чат.",
+	"GAG_UNTIL_MAP_CHANGE": "До смены карты",
+	"UNGAG": "Снять молчанку",
+	"UNGAG_TIP": "Возвращает возможности игроку использовать голосовой и обычный чат.",
+	"FORCE_RANDOM": "Случайное распределение",
+	"FORCE_RANDOM_TIP": "Принудительно перемещает игрока(ов) в случайную команду.",
+	"READY_ROOM": "Комната ожидания",
+	"READY_ROOM_TIP": "Принудительно перемещает игрока(ов) в комнату ожидания.",
+	"SET_TEAM": "Задать команду",
+	"SET_TEAM_TIP": "Принудительно перемещает игрока(ов) в выбранную команду.",
+	"CHANGE_MAP": "Смена карты",
+	"CHANGE_MAP_TIP": "Моментально меняет карту.",
+	"CALL_VOTE": "Голосование выбора карты",
+	"CALL_VOTE_TIP": "Запускает голосование выбора карты.",
+	"PLUGIN": "Плагин",
+	"STATE": "Состояние",
+	"LOAD_PLUGIN": "Загрузить плагин",
+	"UNLOAD_PLUGIN": "Выгрузить плагин",
+	"UNLOAD_PLUGIN_TIP": "Временно выгружает плагин.",
+	"UNLOAD_PLUGIN_SAVE_TIP": "Сохранить в настройках отключение плагина.",
+	"LOAD_PLUGIN_TIP": "Временно загружает плагин.",
+	"LOAD_PLUGIN_SAVE_TIP": "Сохранить в настройках включение плагина.",
+	"RELOAD_PLUGIN": "Перезагрузка плагина",
+	"ALLTALK_ENABLED": "Включить межкомандное общение.",
+	"ALLTALK_DISABLED": "Выключить межкомандное общение.",
+	"CHEATS_TOGGLED": "{Enabled:BoolToPhrase:enabled|disabled} читы.",
+	"ALL_TALK_TAG": "[Межкомандное общение]",
+	"ALLTALK_TOGGLED": "{Enabled:BoolToPhrase:enabled|disabled} межкомандное общение.",
+	"ALLTALK_PREGAME_TOGGLED": "{Enabled:BoolToPhrase:enabled|disabled} межкомандное общение в режиме пред-игры.",
+	"ALLTALK_LOCAL_TOGGLED": "{Enabled:BoolToPhrase:enabled|disabled} локальное межкомандное общение",
+	"ALLTALK_NOTIFY_ENABLED": "Межкомандное общение включено.",
+	"ALLTALK_NOTIFY_DISABLED": "Межкомандное общение выключено.",
+	"ALLTALK_PREGAME_NOTIFY_ENABLED": "Межкомандное общение включено в режиме пред-игры.",
+	"ALLTALK_PREGAME_NOTIFY_DISABLED": "Межкомандное общение выключено в режиме пред-игры.",
+	"ALLTALK_LOCAL_NOTIFY_ENABLED": "Локальное межкомандное общение включено.",
+	"ALLTALK_LOCAL_NOTIFY_DISABLED": "Локальное межкомандное общение выключено.",
+	"ALL_TALK_LOCAL_DESCRIPTION": "Включить локальное межкомандное общение.",
+	"CLIENT_KICKED": "выкинут с сервера {TargetName}.",
+	"CLIENT_KICKED_REASON": "выкинут с сервера {TargetName} ({Reason}).",
+	"FRIENDLY_FIRE_SCALE": "задать процент урона для дружественного огня {Scale:Format:%.2f}.",
+	"FRIENDLY_FIRE_ENABLED": "Дружественного огня включён.",
+	"FRIENDLY_FIRE_DISABLED": "Дружественного огня выключен.",
+	"FF_TAG": "[Дружественный огнь]",
+	"RESET_GAME": "сбросить игру.",
+	"CHANGE_TEAM": "перемещены {TargetCount} {TargetCount:Pluralise:player|players} в {Team:TeamName::Lower}.",
+	"RANDOM_TEAM": "перемещен(ы) {TargetCount} {TargetCount:Pluralise:player|players} в случайную(ые) команду(ы).",
```

Comment:
As above, the `{TargetCount:Pluralise:player|players}` will show `player` or `players` depending on if the `TargetCount` value is more than 1 or not, so `player|players` needs translating.